src/pages/workspace/distanceRates/PolicyDistanceRatesPage.tsx

```diff
@@ -60,10 +60,11 @@ function PolicyDistanceRatesPage({policy, route}: PolicyDistanceRatesPageProps)
         [policy?.customUnits],
     );
     const customUnitRates: Record<string, Rate> = useMemo(() => customUnit?.rates ?? {}, [customUnit]);
-    const canDeleteSelectedRates = selectedDistanceRates.length !== Object.values(customUnitRates).length;
-    const canDisableSelectedRates = Object.values(customUnitRates)
-        .filter((rate: Rate) => !selectedDistanceRates.includes(rate))
-        .some((rate) => rate.enabled);
+    // Filter out rates that will be deleted
+    const allSelectableRates = Object.values(customUnitRates).filter((rate) => rate.pendingAction !== CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE);
+    const canDisableOrDeleteSelectedRates = allSelectableRates
+        .filter((rate: Rate) => !selectedDistanceRates.some((selectedRate) => selectedRate.customUnitRateID === rate.customUnitRateID))
+        .some((rate) => rate.enabled === true);
```

Comment:
Are there some other truthy values that can be stored in the `rate.enabled`? do we need the equal to true?

Reply:
uhmm no, you're right, I will revert this change